### PR TITLE
feat(kms): add b1 tracking

### PR DESCRIPTION
**Pull request by rjamet-ovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-14226
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

Add tracking on batch 1 perimeter

### CODE REVIEW

**Review by anooparveti — DISMISSED** [review]

**Review by vovh — APPROVED** [review]

**Review by pauldkn — DISMISSED** [review]

File: packages/manager/apps/key-management-service/src/pages/listing/index.tsx [hunk 1]
```diff
@@ -84,45 +91,53 @@ export default function Listing() {
...
-          {tError('manager_error_page_default')}
-        </OsdsMessage>
-      )}
-
-      {isLoading && !error && (
-        <div className="text-center">
-          <OsdsSpinner inline size={ODS_SPINNER_SIZE.md} />
+    <Suspense fallback={<Loading />}>
+      <BaseLayout header={headerProps} message={<Notifications />}>
+        <div className={'flex mb-3 mt-6'}>
```
Comment (tibs245) [inline packages/manager/apps/key-management-service/src/pages/listing/index.tsx:96]:
```suggestion
        <div className='flex mb-3 mt-6'>
```

File: packages/manager/apps/key-management-service/src/pages/listing/index.tsx [hunk 2]
```diff
@@ -84,45 +91,53 @@ export default function Listing() {
...
+          <div className={'mt-8'}>
+            <Datagrid
+              columns={columns}
+              items={okms || []}
+              totalItems={0}
+              sorting={sorting}
+              onSortChange={setSorting}
+              contentAlignLeft
+            />
+          </div>
```
Comment (tibs245) [inline packages/manager/apps/key-management-service/src/pages/listing/index.tsx:137]:
```suggestion
        {isLoading && !error && (
        <>
          <div className="text-center">
            <OsdsSpinner inline size={ODS_SPINNER_SIZE.md} />
          </div>
          <div className={'mt-8'}>
            <Datagrid
              columns={columns}
              items={okms || []}
              totalItems={0}
              sorting={sorting}
              onSortChange={setSorting}
              contentAlignLeft
            />
          </div>
          </>
```

**Review by tibs245 — APPROVED** [review]

File: packages/manager/apps/key-management-service/src/pages/listing/terminate/TerminateKms.tsx [hunk 3]
```diff
@@ -13,8 +15,20 @@ export default function TerminateKms() {
...
+        pageName: 'delete_kms_success',
+      });
+      closeModal();
+    },
+    onError: () => {
+      trackPage({
+        pageType: PageType.bannerError,
+        pageName: 'delete_kms_error',
+      });
+      closeModal();
```
Comment (chipp972) [inline packages/manager/apps/key-management-service/src/pages/listing/terminate/TerminateKms.tsx:30]: You also close the modal on error ?
You don't display the error on the modal ?

**Review by chipp972 — APPROVED** [review]

File: packages/manager/apps/key-management-service/src/pages/listing/terminate/TerminateKms.tsx (hunk 3 above)
Comment (rjamet-ovh) [reply packages/manager/apps/key-management-service/src/pages/listing/terminate/TerminateKms.tsx:30]: Ee handled error on the previous page (dashboard, list).
In any case, this modal will be replaced by the react-manager-component `<DeleteModal />`, so we'll fixed that.

File: packages/manager/apps/key-management-service/src/pages/listing/index.tsx (hunk 1 above)
Comment (rjamet-ovh) [reply packages/manager/apps/key-management-service/src/pages/listing/index.tsx:96]: will be fixed on upcoming refacto

**Review by anooparveti — APPROVED** [review]

**Review by pauldkn — APPROVED** [review]

**Review by tibs245 — APPROVED** [review]

**Pull request merged**